index.html

```diff
@@ -8,34 +8,204 @@
+                <h1 class="tonic">Tonic</h1>
+                <div class="grid-container">
+                    <div class="grid-item1">CANOPY</div>
+                    <div class="grid-item2"><img src="image/Counter.svg" alt="Counter"></div>
+                    <div class="grid-item3">Back End Dev</div>
+                    <div class="grid-item4"><img src="image/Counter.svg" alt="Counter"></div>
+                    <div class="grid-item5">2015</div>
+                </div>
+                <article>
+                    <p class="canopy">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </p>
+                </article>
+                <div class="languages">
+                    <div class="lang1"><img src="image/Tag_html.svg" alt="HTML Tag"></div>
+                    <div class="lang2"><img src="image/Tag_css.svg" alt="HTML Tag"></div>
+                    <div class="lang3"><img src="image/Tag.svg" alt="HTML Tag"></div>
+                </div> 
+                <div class="nav-button">
+                    <a href="https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/images/m2_mobile/works-mobile.png"><img src="image/Enabled.svg" alt="See Project button"></a> 
+                </div>    
+            </div>
+        </section>
+
+        <section>
+            <div class="border_project3">
+                <div class="image_availability">
+                    <img src="image/Availability.svg" alt="Snapshoot Portfolio Availability">
+                </div>
+                <header class="stories"><img src="image/Project Title.svg" alt="Project Title">
+                </header>
+                <div class="grid-container">
+                    <div class="grid-item1">CANOPY</div>
+                    <div class="grid-item2"><img src="image/Counter.svg" alt="Counter"></div>
+                    <div class="grid-item3">Back End Dev</div>
+                    <div class="grid-item4"><img src="image/Counter.svg" alt="Counter"></div>
+                    <div class="grid-item5">2015</div>
+                </div>
+                <article>
+                    <p class="canopy">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </p>
+                </article>
+                <div class="languages">
+                    <div class="lang1"><img src="image/Tag_html.svg" alt="HTML Tag"></div>
+                    <div class="lang2"><img src="image/Tag_css.svg" alt="HTML Tag"></div>
+                    <div class="lang3"><img src="image/Tag.svg" alt="HTML Tag"></div>
+                </div> 
+                <div class="nav-button">
+                    <a href="https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/images/m2_mobile/works-mobile.png"><img src="image/Enabled.svg" alt="See Project button"></a> 
+                </div> 
```

Comment:
- [ ] Kindly change the image used as a button and use a button tag with its text based on the template you choose.
- [ ] Your design missed some alignment and background issues. Please kindly fix the issue :wrench: 

***Template design***
![Screenshot from 2022-12-02 14-10-41](https://user-images.githubusercontent.com/65129228/205435717-950246e4-b3e2-41e4-b136-d55bdeacf1ce.png)



***Your design***

![Screenshot from 2022-12-03 11-55-52](https://user-images.githubusercontent.com/65129228/205435734-6bee332a-2d9f-4344-abbe-f76b5fd45d89.png)